presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -55,6 +55,10 @@
     @Named("databases.presto.server_address")
     private String serverAddress;
 
+    @Inject
+    @Named("databases.presto.jdbc_user")
+    private String jdbcUser;
```

Comment:
yes, it is required for PrestoCLITests to pass in case we set hive.security option by default.